.github/workflows/conformance-datapath.yaml

```diff
@@ -227,10 +227,12 @@ jobs:
             --nodes-without-cilium=kind-worker3 \
             --helm-set-string=kubeProxyReplacement=${{ matrix.kpr }} \
             --helm-set=bpf.masquerade=false"
-          TUNNEL="--helm-set-string=tunnel=vxlan"
```

Comment:
probably easier to follow:

```yaml
TUNNEL="--helm-set-string=tunnel=${{ matrix.tunnel }}"
if [ "${{ matrix.tunnel }}" == "disabled" ]; then
    TUNNEL="--helm-set-string=autoDirectNodeRoutes=true --helm-set-string=ipv4NativeRoutingCIDR=10.244.0.0/16 --helm-set-string=ipv6NativeRoutingCIDR=fd00:10:244::/56"
fi
```

Reply:
Not sure what I was thinking :facepalm: Thanks!